src/devices/imhotepcreation.ts

```diff
@@ -0,0 +1,156 @@
+import tz from '../converters/toZigbee';
+import fz from '../converters/fromZigbee';
+import * as exposes from '../lib/exposes';
+import * as constants from '../lib/constants';
+import * as reporting from '../lib/reporting';
+import {Zh, Definition} from '../lib/types';
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const definitions: Definition[] = [
+    {
+        zigbeeModel: ['E-Ctrl', 'RPH E-Ctrl', 'RSS E-Ctrl'],
+        model: 'E-Ctrl',
+        vendor: 'Imhotep Creation',
+        description: 'Heater thermostat PH25 and compliant',
+        whiteLabel: [
+            {vendor: 'Imhotep Creation', model: 'RSS E-Ctrl', description: 'Towel heater thermostat THIE (TH ECTRL) and compliant',
+                fingerprint: [{modelID: 'RSS E-Ctrl'}]},
+            {vendor: 'Imhotep Creation', model: 'RPH E-Ctrl', description: 'Panel radiant heater thermostat MPHIE (NRPH) and compliant',
+                fingerprint: [{modelID: 'RPH E-Ctrl'}]},
```

Comment:
I missed the whitelabels here, could you add the pictures for it back in https://github.com/Koenkk/zigbee2mqtt.io/pull/2331 ?